src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -28,7 +28,6 @@ const FormFieldControlErrorWrapper = styled(Box)`
 const FormFieldControlStyled = styled(FormFieldControl)`
   display: flex;
   flex-direction: column;
-  align-items: flex-end;
```

Comment:
🤔

Reply:
https://github.com/getsentry/sentry/pull/6915 introduced a regression (removing this also causes another one, specifically when form fields have `inline=true`).

Reply:
🤔 they both cause a regression?

Reply:
Yeah look at percy-storybook + 
https://percy.io/getsentry/sentry-storybook/builds/469815